index.html

```diff
@@ -2399,6 +2399,36 @@ <h4>Not Mapped</h4>
 					<span class="property not-mapped"><a href="#not_mapped">Not mapped*</a></span>
 				</td>
 			</tr>
+			<tr id="ariaBraillelabel">
+				<th><a class="property-reference" href="#aria-braillelabel"><code>aria-braillelabel</code></a></th>
+				<td class="attr-msaa-ia2">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.braillelabel</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleLabel</span>
```

Comment:
@cookiecrook are the Mac mappings correct?

Reply:
Yes https://github.com/WebKit/WebKit/blob/main/Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm#L496